safehttp/incoming_request.go

```diff
@@ -108,3 +109,23 @@ func (r *IncomingRequest) MultipartForm(maxMemory int64) (*MultipartForm, error)
 			}},
 		nil
 }
+
+// Context returns the underlying http.Request context for a
+// safehttp.IncomingRequest. This is always non-nil and will default to the
+// background context.
```

Comment:
`Context returns the context of the IncomingRequest. This is alwas non-nil ...`
The context of the `IncomingRequest` is the context of the underlying request.